File: 6-kyu/exclamation-marks-series-#17--put-the-exclamation-marks-and-question-marks-to-the-balance,-are-they-balanced-/python/solution.py
def balance(left, right):
    lft = left.count('!') * 2 + left.count('?') * 3
    rgt = right.count('!') * 2 + right.count('?') * 3
    return "Left" if lft > rgt else "Balance" if lft == rgt else "Right"
  

File: 6-kyu/exclamation-marks-series-#17--put-the-exclamation-marks-and-question-marks-to-the-balance,-are-they-balanced-/javascript/solution.js
const balance = (left, right) => {
  const count = (str, toCount) => [...str].filter(elem => elem === toCount).length;
  const leftCount = count(left, "!") * 2 + count(left, "?") * 3;
  const rightCount = count(right, "!") * 2 + count(right, "?") * 3;
  return leftCount > rightCount ? "Left" : leftCount === rightCount ? "Balance" : "Right";
};
